### ISSUE TITLE
Insert ALT format

### ISSUE THREAD

**Issue by awgymer** [KolmogorovLab/Severus#19]
I notice some inserts with an ALT format like `<chr>:<start>-<stop>` (i.e. `chr8:141415620-141415787`) and as far as I can tell this is not VCF spec. This causes downstream tools to throw errors. 

Where do these format `INS` come from and is it possible to have them replaced by a spec format?

**Comment by aysegokce** [comment]
Hello,
That is an older representation of the tandem duplications in VNTRs. To unify the representation of indels in VNTRs, we convert duplications to insertions (since in some reads, it may be represented as duplication while in others as an insertion). 

In the current version, it is replaced with `<INS>` in the ALT column and `ALINGED_POS` in the INFO column. 

Please let me know if this representation works.
Ayse

**Comment by awgymer** [comment]
Testing with the same sample on v1.0 returns the line 
```
chr8    141415620       severus_INS20553        N       <DUP>   60.0    PASS    PRECISE;SVTYPE=INS;SVLEN=118;INSIDE_VNTR=TRUE;MAPQ=60.0 GT:GQ:VAF:hVAF:DR:DV    0/0:204:0.21:1.00,0.00,0.00:22:6
```

This appears to be slightly different from what you have outlined here?

**Comment by aysegokce** [comment]
In this case, it is a tandem duplication of a VNTR as `INSIDE_VNTR=TRUE ` suggests. For those cases, we change the ALT column to `<DUP>`. 

We use the `ALINGED_POS `representation for insertions with known alignments (other than duplications) as in the example below:
`chr16   71242755        severus_INS13587        N       CTACTCCACATTTTCTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGGATTCTTATATTTTGCTTTATTCCAAATTATAGCCACTAAGATTCGTATCACTTTATAGTTTCTCAGGAAAAAGCACTTCCTTTCATATAGACAATTTTAGAAAAGTTTCTAAATTACAATATACCTTAATTTTTGAAATTGCATACCAGTTTTTTTCATCATTATAATCAAAATATAAAGATCCTTCTGAACTATGATAATGTGATAACTGATATAATCTATCAGAGTTACATATGATTTCAAGGTGAAATGCTGTTTTTCATAATCTGTGTTCATGATTTTTGAATGTTGATGCATTCTTGCCTTCCTGGAATAAACACTTAGTCATAATGTATAGTTTTTTTAAAAAAAAATACCCCAGTGACTTTGATTTTCTAATATTTTATTTTGGGTTTTTTACATCTGTAGTCCTAAGTGAGATTGAACTATTAATTGTCTTTGCTTCCCTTGGCCAGTTTTGTGAAGGAATAAACG        60.0    PASS    PRECISE;SVTYPE=INS;SVLEN=520;ALINGED_POS=chr3:123876019-123876539;MAPQ=60.0;PHASESETID=68978924;HP=2   GT:GQ:VAF:hVAF:DR:DV    0/1:514:0.39:0.00,0.00,1.00:126:80`

Sorry for the confusion. 
Ayse

**Comment by aysegokce** [comment]
Thank you for the feedback. The vcf output is updated in version 1.1

**Comment by awgymer** [comment]
@aysegokce I have been testing the newer output format, one issue I have noticed is that - as the above example shows - a row with and `ALT` of `<DUP>` still has an `SVTYPE=INS` in `INFO`. Is this deliberate? If so is there a reason not to set it to `DUP`?

**Comment by minw2828** [comment]
Hello @aysegokce, the latest VCF specification (version 4.5) which was released on 28 Jun 2024, has section 5.7 devoted to STR and VNTR. https://github.com/samtools/hts-specs/blob/master/VCFv4.5.pdf

Would you consider aligning the Severus output format with VCFv4.5 in a future release? This would help streamline downstream tools. Thank you!

**Comment by minw2828** [comment]
Could @aysegokce provide a reference on the format (such as chr8:141415620-141415787) being an older representation of the tandem duplications in VNTRs, please? 

I want to have a look at it as I am not familiar with variable number tandem repeats (VNTRs). 

Thank you!

**Comment by aysegokce** [comment]
Hello @awgymer,
Can you please confirm it is with v1.1? If so, please share the line from the vcf and an IGV screenshot.
Best
Ayse

**Comment by awgymer** [comment]
This appears resolved in 1.1. (All `<DUP>` now `SVTYPE=DUP`)
